README.md

```diff
@@ -34,40 +37,41 @@ git clone https://github.com/Azure-Samples/functions-quickstart-javascript-azd.g
 ## Run on your local environment
 
 ### Prepare your local environment
+1) Navigate to the new _functions-quickstart-javascript-azd_ folder.
```

Comment:
```suggestion
1) Navigate to the new _functions-quickstart-javascript-azd/src/function_ folder.
```

I notice we have to be careful throughout because the function really lives in <reporoot>/src/functions

Consider telling them to CD into the right folder right after the clone on L34:

```bash
git clone https://github.com/Azure-Samples/functions-quickstart-javascript-azd.git
cd ./functions-quickstart-javascript-azd/src/function
```